npm-shrinkwrap.json

```diff
@@ -1,574 +0,0 @@
-{
```

Comment:
If Sails loads package A, and package A does not lock down its dependency versions, removing this file means package B will float to the latest available version. I'd rather not introduce the possibility of breaking in that way.